react/src/Header.js

```diff
@@ -97,7 +98,7 @@ const Header = (props) => {
             <div style={{width:"100%",background:titleBackgroundColor}}>
             <table width="100%" cellPadding="0" cellSpacing="0"><tbody>
             <tr title={"App Deployed:" + header.app?.deployed + " | App Launched: " + header.app?.launched + " | Page Loaded: " + header.page?.loaded}>
-                <td width="33%" style={{paddingLeft:"2pt",whiteSpace:"nowrap"}}>
+                <td width="38%" style={{paddingLeft:"2pt",whiteSpace:"nowrap"}}>
```

Comment:
Do you not want a minimum width for this kind of thing? At some point, 30% can be pretty small. Similar comment for other columns. I won't mark them separately.

Reply:
This was slightly tricker to get right WRT centering, because of the left and right content. I'm sure I'm probably doing it wrong CSS-wise. I added TODO comment to clean this up.